### ISSUE TITLE
Icon request: fa-twitter-circle, fa-facebook-circle (circle social icons)

### ISSUE THREAD

**Issue by jhoffmcd** [FortAwesome/Font-Awesome#4167]
I've had a few instances where the design of a site calls for and repetition of circular shapes, including the look of social media links.

You see this all the time in image based icon sets, it would be nice to see if in Font-Awesome as well.

fa-facebook-circle would be just like fa-facebook-square, just circular.

Here is an image of an icon set that is similar.

![image](https://cloud.githubusercontent.com/assets/2348095/3961988/90becb04-2760-11e4-9a2c-9abf8d98170b.png)

I know some logo's probably won't translate due to size, but i can see a few working.

**Comment by ivanporras** [comment]
thank you so much

**Comment by gopalkumar315** [comment]
thanks a lot man

**Comment by kksrini89** [comment]
@pakimin Really awesome. Thanks

**Comment by leepettijohn** [comment]
I agree CSS can assist.  The only issue is when you want the symbol to be transparent.

**Comment by rahulsemwal** [comment]
Thanks @pakimin

**Comment by Wolfneo** [comment]
Hi,
I do not find these logos of social networks in the circle on Font-awesome, have they not been added to the library?
Thank you

**Comment by raj1981** [comment]
# hfhoafa

**Comment by afayes** [comment]
I wish there was  a way to vote for this issue. I came here because I googled for font awesome circular face book icon. I am not a designer but a backend developer. While the above CSS may do the job, it is definitely much easier to have it natively supported out of the box.

**Comment by pakimin** [comment]
@afayes Well you can combine the needed icon with icon fa-circle..
then you need to set both :before and :after and play with the font-sizes to fit. But then again, you need to use positioning with absolute and a lot of more work to do.

Frontend and Backend developers always have something to argue, but at the end they always need each other.

**Comment by afayes** [comment]
@pakimin there are solutions like yours that would work. But like you mentioned in your example there is 'a lot of more work to do'. I see this issue as a feature request that will allow users to easily use social icons with a circle out of the box. Based on a google search there seems to be a demand for it. Every owner of a framework wants to make things as easy as possible for their users. By implementing features in demand, font awesome is listening to their users needs.

**Comment by gnievap** [comment]
Thanks!

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
